tests/php/SiteTreeSubsitesTest.php

```diff
@@ -320,14 +378,14 @@ class SiteTreeSubsitesTest_ClassB extends SiteTree implements TestOnly
 
 class SiteTreeSubsitesTest_ErrorPage extends ErrorPage implements TestOnly
```

Comment:
Ideally we'd move these into a `SiteTreeSubsitesTest` folder and call them `StubErorrPage` or something